app/subscribers/super_good/solidus_taxjar/spree/reporting_subscriber.rb

```diff
@@ -3,12 +3,32 @@ module SolidusTaxjar
     module Spree
       module ReportingSubscriber
         include ::Spree::Event::Subscriber
+
         event_action :report_transaction, event_name: :shipment_shipped
+        event_action :replace_transaction, event_name: :order_recalculated
 
         def report_transaction(event)
           return unless SuperGood::SolidusTaxjar.reporting_enabled
+
           SuperGood::SolidusTaxjar::ReportTransactionJob.perform_later(event.payload[:shipment].order)
         end
+
+        def replace_transaction(event)
+          order = event.payload[:order]
+
+          return unless SuperGood::SolidusTaxjar.reporting_enabled
+
+          if order.complete? && order.paid? && amount_changed?(order)
+            SuperGood::SolidusTaxjar::ReplaceTransactionJob.perform_later(event.payload[:order])
+          end
+        end
+
+        private
+
+        def amount_changed?(order)
+          SuperGood::SolidusTaxjar.api.show_latest_transaction_for(order).amount !=
```

Comment:
It's possible that the `amount + sales_tax` could equal the `order.total` for the wrong reason.

We do not want this method to return true just because the sales tax changed, for example.

Perhaps we can reconsider this in a future PR?

Reply:
Mhmm that's an interesting point! So if the sales tax changes due to a re-calcalculation, we **dont** want to report the changed sales tax to Taxjar?

This is an open question I've honestly never thought about it. Down to move this to a discussion thread.

Reply:
I captured this as a new issue: https://github.com/SuperGoodSoft/solidus_taxjar/issues/166